andela/nazgul-cfh
# #156100325 Start game and save game history

#### What does this PR do?
This PR makes it possible to easily start the game and then create a game history/log
#### Description of Task to be completed?
Seed the remote database with questions answers
#### How should this be manually tested?

clone the repository
`cd` into the cloned directory
run `npm install` to update the node packages
run `npm start` next to start the application.
when the application starts navigate to the signup or login page at `https://nazgul-cfh-staging.herokuapp.com/#!/signup` or `https://nazgul-cfh-staging.herokuapp.com/#!/signin` respectively.
signup or login and then navigate to the game page by clicking `play with strangers` when you are successfully signed in.
ensure that you are up to the minimum number of 3 players before starting the game.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#156100325
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
